### PR TITLE
Add link to membership

### PR DESCRIPTION
**Pull request by tzhelyazkova**
T188254
Changes made to the icomoonwikimedia font are in relation to the new icon for external link.
Description of how to add new icons is added for clarity.

### CODE REVIEW

File: skins/cat17/src/sass/layouts/_membership.scss [hunk 1]
```diff
@@ -203,4 +199,17 @@
...
 	}
 }
+.external-link {
+	margin-top: 90px;
+	@include themify($themes) {
+		border-bottom: themed('border1pxSecondary');
+	}
+}
+.side {
+	diplsay: inline;
```
Comment (wiese) [inline skins/cat17/src/sass/layouts/_membership.scss]: This is a typo, I guess.

File: skins/cat17/templates/Membership_Application.html.twig [hunk 2]
```diff
@@ -45,6 +45,14 @@
 							'showActiveMembershipBenefits': showMembershipTypeOption == 'true'
 						} only %}
 					</div>
+					<div class="col-xs-12 col-md-4">
+						<div class="external-link">
+							<a href="https://wikimedia.de/wiki/Mitgliedschaft" target="_blank">
```
Comment (wiese) [inline skins/cat17/templates/Membership_Application.html.twig]: Not wrong, but I'd make it "configurable" in the content repo (dedicated key).

File: composer.lock [hunk 3]
```diff
@@ -1789,16 +1789,16 @@
         },
         {
             "name": "silex/silex",
-            "version": "v2.2.2",
+            "version": "v2.2.4",
```
Comment (wiese) [inline composer.lock]: I don't think that such updates should happen as a by-product of a, AFAICT, completely unreleated front-end story. Accidentally committed?

File: skins/cat17/src/sass/main.scss [hunk 4]
```diff
@@ -3,7 +3,22 @@
 @import "jcf/dist/css/theme-minimal/jcf";
 @import "bootstrap-sass/assets/stylesheets/bootstrap";
 
+/**
+ * Default font "style" file as @generated by icomoon. If you ever want to change it
+ * - go to icomoon.org
```
Comment (wiese) [inline skins/cat17/src/sass/main.scss]: icomoon.io - where did my comment go?

File: skins/cat17/templates/Membership_Application.html.twig (hunk 2 above)
Comment (gbirke) [reply skins/cat17/templates/Membership_Application.html.twig]: Originally, I advised Tonina against configurability because I imagined the link to be only in one place and putting it in `app/config` would have felt wrong to me. But since it will also appear on the donation confirmation page and could hypothetically have an English version, I've changed my mind and also think that we could put this into `messages.json`.

File: skins/cat17/src/sass/components/fonts/icomoonwikimedia/style.scss [hunk 5]
```diff
@@ -20,170 +20,172 @@
...
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
 
-.icon-payment-sofort {
-  &:before {
-    content: $icon-payment-sofort;
-  }
+@function unicode($str) {
+  @return unquote("\"") + $str + unquote("\"");
```
Comment (gbirke) [inline skins/cat17/src/sass/components/fonts/icomoonwikimedia/style.scss]: I don't understand why this change is necessary, could you put in a comment here?

Comment (tzhelyazkova) [reply skins/cat17/src/sass/components/fonts/icomoonwikimedia/style.scss]: The comment in main.scss  mentions that style.scss is generated by icomoon. I'm not sure what comment exactly to add to this line here.
